planner/core/plan_stats.go

```diff
@@ -85,6 +90,9 @@ func RequestLoadStats(ctx sessionctx.Context, neededHistItems []model.TableItemI
 	var timeout = time.Duration(waitTime)
 	err := domain.GetDomain(ctx).StatsHandle().SendLoadRequests(stmtCtx, neededHistItems, timeout)
 	if err != nil {
+		_, digest := stmtCtx.SQLDigest()
+		logutil.BgLogger().Warn("SendLoadRequests failed",
+			zap.String("digest", digest.String()), zap.Error(err))
```

Comment:
What about write this warn message to slow logs or statement summary?

Reply:
I don't think record error detail in slowlog is a good choice, but I think it's proper to record a flag if sync stats failed in slow log.